locale/en_US/locale.xml

```diff
@@ -216,8 +216,8 @@
 	<message key="user.role.managers">Journal Managers</message>
 	<message key="user.role.subEditors">Section Editors</message>
 	<message key="user.role.subscriptionManagers">Subscription Managers</message>
-	<message key="user.role.journalAssistant">Journal Assistant</message>
-	<message key="user.role.journalAssistants">Journal Assistants</message>
+	<message key="user.role.journalAssistant">Assistant</message>
+	<message key="user.role.journalAssistants">Assistants</message>
```

Comment:
This will then maybe be removed to pkp-lib, when OMP changes come? And the keys below as well?

Reply:
I initially though to do that, but OMP doesn't support a multi-context site admin area, does it? In which case, I don't think this locale string is used...

Reply:
Ah yes... forgot about that :-P